conf/glacier-user.conf

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+    <policy user="nemo">
+        <allow send_destination="net.connman"/>
+        <allow send_destination="com.nokia.dsme" send_interface="com.nokia.dsme.request" send_member="req_shutdown"/>
+        <allow send_destination="com.nokia.dsme" send_interface="com.nokia.dsme.request" send_member="req_reboot"/>
+        <allow send_destination="com.nokia.dsme" send_interface="com.nokia.dsme.request" send_member="req_powerup"/>
```

Comment:
Out of curiosity: Was there some particular reason for these tweaks to allow nemo user to do reboot/shutdown etc?

Reply:
becouse only root or console user can do that

Reply:
Well, that is how it more or less is. But is there some reason for making the change? I'd assume it is something along the line "we'd like user to be able to reboot without hassle". But if it is something else, it might be something that needs to be addressed at sailfish side too.

Reply:
sfos use some strange magic and all users in all sessions run from console

Reply:
ack.

NB there is a task about changing how those reboot/shutdown privilege requirements should go. I'll add a not that it needs to be brought to your (nemomobile/glacier) attention too when it is  actually worked on.

Reply:
now in mer bugzilla -> https://bugs.merproject.org/show_bug.cgi?id=2044